libafl_bolts/src/lib.rs

```diff
@@ -341,6 +341,8 @@ pub enum Error {
     InvalidCorpus(String, ErrorBacktrace),
     /// Error specific to a runtime like QEMU or Frida
     Runtime(String, ErrorBacktrace),
+    /// The `Input` was invalid.
+    InvalidInput(ErrorBacktrace),
```

Comment:
Wouldn't it make sense to include a string here? Like, the path of the invalid input, and maybe a root cause if there is any?

Reply:
Especially given the callback is user provided, adding a string will make things way easier to debug IMHO. The callback can differentiate between different invalid input causes